src/client/app/components/UIOptionsComponent.tsx

```diff
@@ -63,6 +64,12 @@ class UIOptionsComponent extends React.Component<UIOptionsPropsWithIntl, UIOptio
 		};
 	}
 
+	public componentDidUpdate(prev: UIOptionsProps) {
+		if (prev.chartToRender !== this.props.chartToRender) {
+			ReactTooltip.rebuild(); // This rebuilds the tooltip so that it detects the marker that disappear because the chart type changes.
```

Comment:
This is a small point but OED is trying to get comments on lines above that line and not on the same line, esp. if there is any length to the comment.